scalafix-core/src/main/scala/scalafix/util/Severity.scala

```diff
@@ -0,0 +1,23 @@
+package scalafix.util
+
+import scalafix.config.ReaderUtil
+
+import metaconfig.Reader
+
+sealed abstract class Severity(color: String, val order: Int)(
+    implicit name: sourcecode.Name)
+    extends Ordered[Severity] {
+  override val toString: String =
+    color + name.value.toLowerCase + Console.RESET
```

Comment:
this kind of thing can break (dumb) terminals that don't understand escape codes.